model/status.go

```diff
@@ -22,7 +22,7 @@ type Status struct {
 	Status         string `json:"status"`
 	Manual         bool   `json:"manual"`
 	LastActivityAt int64  `json:"last_activity_at"`
-	ActiveChannel  string `json:"active_channel" db:"-"`
```

Comment:
This is not returned in APIv3 at all. We should not return `ActiveChannel` to the client since it doesn't need it and because it would be leaking channel IDs